src/views/shortcodes/my-attendance-list.php

```diff
@@ -22,6 +22,9 @@
 					<span class="datetime">(<?php echo $start_date; ?>)</span>
 				<?php endif; ?>
 			</a>
+			<a href="<?php echo esc_url( get_permalink( $id ). 'tickets' ); ?>" target="_blank">
```

Comment:
:no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: 75272948850aa8b4a1a925eadc2ba8d2</sub>

Reply:
```suggestion
			<a href="<?php echo esc_url( get_permalink( $id ) . 'tickets/' ); ?>" target="_blank">
```